core/drivers/corda-driver/.env.docker.template

```diff
@@ -1,6 +1,9 @@
 NETWORK_NAME=<container-name-suffix>
 DRIVER_PORT=<driver-server-port>
+DRIVER_RPC_USERNAME=driverUser1
+DRIVER_RPC_PASSWORD=test
 EXTERNAL_NETWORK=<docker-bridge-network>
-DOCKER_IMAGE_NAME=weaver-corda-driver
+DOCKER_IMAGE_NAME=ghcr.io/hyperledger-labs/weaver-corda-driver
```

Comment:
Perhaps we should retain `ghcr.io` in `DOCKER_REGISTRY` and the rest in `DOCKER_IMAGE_NAME`?